kloppy/infra/serializers/event/statsbomb/deserializer.py

```diff
@@ -416,6 +432,57 @@ def _parse_take_on(take_on_dict: Dict) -> Dict:
     }
 
 
+def _parse_duel(
+    raw_event: dict,
+    event_type: int,
+) -> Dict:
+    duel_dict = None
+    duel_qualifiers = []
+
+    if event_type == SB_EVENT_TYPE_DUEL:
+        duel_dict = raw_event.get("duel", {})
+        type_id = duel_dict.get("type", {}).get("id")
+        if type_id == SB_EVENT_TYPE_AERIAL_LOST:
+            duel_qualifiers = [
+                DuelQualifier(value=DuelType.LOOSE_BALL),
+                DuelQualifier(value=DuelType.AERIAL),
+            ]
+        elif type_id == SB_EVENT_TYPE_TACKLE:
+            duel_qualifiers = [DuelQualifier(value=DuelType.GROUND)]
+    elif event_type == SB_EVENT_TYPE_50_50:
+        duel_dict = raw_event.get("50_50", {})
+        duel_qualifiers = [
+            DuelQualifier(value=DuelType.LOOSE_BALL),
+            DuelQualifier(value=DuelType.GROUND),
+        ]
+
+    qualifiers = duel_qualifiers + _get_body_part_qualifiers(duel_dict)
+
+    outcome_name = duel_dict.get("outcome", {}).get("name") or duel_dict.get(
+        "type", {}
+    ).get("name")
+
+    if outcome_name in DUEL_WON_NAMES:
+        result = DuelResult.WON
+    else:
+        result = DuelResult.LOST
+
+    return {"result": result, "qualifiers": qualifiers}
```

Comment:
I would do similar as above using `DuelResult.WON if outcome_name in DUEL_WON_NAMES else DuelResult.LOST`